dawgz/schedulers.py

```diff
@@ -398,7 +398,10 @@ async def exec(self, job: Job) -> Any:
         ]
 
         if job.array is not None:
-            lines.append("#SBATCH --array=" + comma_separated(job.array))
+            arrayline = "#SBATCH --array=" + comma_separated(job.array)
+            if "simtasks" in job.settings:
+                arrayline += f"%{job.settings['simtasks']}"
+            lines.append(arrayline)
```

Comment:
I think the name `maxsim` is more appropriate.

```suggestion
            line = "#SBATCH --array=" + comma_separated(job.array)
            if 'maxsim' in job.settings:
                line += f"%{job.settings.pop('maxsim')}"
            lines.append(line)
```

Reply:
Agree for the name! I was hesitating. :)